src/components/Delegation/DelegationSummary.js

```diff
@@ -257,7 +256,10 @@ class DelegationSummary extends React.Component<Props, State> {
             style={styles.inner}
             refreshControl={
               <RefreshControl
-                onRefresh={fetchAccountState}
+                onRefresh={() => {
+                  fetchUTXOs()
```

Comment:
Here I had forgot to fetch utxos as well when user pulls to update

Reply:
is it okay not to block on these calls? I assume yes but just want to confirm you didn't forget to await

Reply:
yes they can be async

Reply:
I was going to ask the same thing. That I thought that maybe this could be an await. Maybe the issues with the loading dialog could be related to multiple calls turning it off / on?

Reply:
No these are the call generated then user pulls to refresh, in which case the spinner works just fine for both Android and iOS.

It just doesn't work in the initial loading, in which case the calls are triggered inside a `ComponentDidMount` or after the component is focused.

Reply:
Okay maybe in that case you can do something like
```
onRefresh={() => Promise.all(fetchUTXOs(), fetchAccountState())}
```
should behave exactly the same way but it won't be considered an error if ever `onRefresh` is refactored to have special handling for async functions

Reply:
Ok I added this too (it was failing but it's because `Promise.all` requires an iterable)